modules/sqs/main.tf

```diff
@@ -46,7 +46,7 @@ resource "aws_sqs_queue" "metadata_queue" {
   message_retention_seconds   = var.metadata_queue_message_retention_time_seconds
   tags                        = local.tags
   policy                      = data.aws_iam_policy_document.metadata_queue_policy.json
-  visibility_timeout_seconds  = 300
+  visibility_timeout_seconds  = 900 # Set to the lambda max time
```

Comment:
should this be changed into a variable under `sqs/variables.tf`?

Reply:
I'm matching https://github.com/nasa/cumulus-orca/blob/022af93b6fc07414a2b977107c0f38741e9ede59/modules/sqs/main.tf

Reply:
okay. I was suggesting to make it a variable under sqs/variables.tf and then use here as `var.visibility_timeout_seconds  `  instead of hardcoding in main.tf